utils/config/daos_server.yml

```diff
@@ -200,11 +200,15 @@
 #
 ## Example: (2 engines * (16 targets/engine * 1GiB)) / 2MiB hugepage size = 16834
 #
+## default: 0
```

Comment:
Maybe some comments on this parameter are still needed to inform the enduser of its role and its allowed values.

Reply:
@Michael-Hennecke recently updated these comments regarding nr_hugepages. will leave him to decide if they should be added to.

Reply:
OK